tests/xharness/Jenkins.cs

```diff
@@ -101,8 +102,24 @@ Task LoadAsync (ref Log log, ILoadAsync loadable, string name)
 					capturedLog.WriteLine ("Failed to load:");
 					capturedLog.WriteLine (v.Exception);
 					capturedLog.Description = $"{name} Listing {v.Exception.Message})";
-				} else if (v.IsCompleted) {
-					capturedLog.Description = $"{name} Listing (ok)";
+				} else if (v.IsCompleted) { // TODO
```

Comment:
TODO?

Reply:
I was using it to navigate, will remove.